torch/csrc/jit/backends/nnapi/nnapi_backend_preprocess.cpp

```diff
@@ -0,0 +1,48 @@
+#include <pybind11/pybind11.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_preprocess.h>
+#include <torch/csrc/jit/python/pybind_utils.h>
+
+namespace py = pybind11;
+
+// Converts model to nnapi and serializes it for mobile
```

Comment:
nit: at least in docs we should say Android NNAPI.

Most people won't be familiar with what nnapi means.